tests/fast-integration/audit-log/client.js

```diff
@@ -0,0 +1,74 @@
+const axios = require("axios");
+const moment = require('moment');
+const {
+    getEnvOrThrow,
+} = require("../utils");
+
+class AuditLogCreds {
+    static fromEnv() {
+        return JSON.parse(getEnvOrThrow("AL_SERVICE_KEY"))
```

Comment:
I see from the code that this env is something more than just a simple string value. Can you briefly describe it in the readme, what is the structure of this JSON?